BOJ_10773.java

```diff
@@ -0,0 +1,30 @@
+package algorithm;
+
+import java.io.*;
+import java.util.Stack;
+
+public class Boj_10773 {
+    public static void main(String[] args) throws IOException {
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        int N = Integer.parseInt(reader.readLine());
+        Stack<Integer> S = new Stack<>();
+        for (int i=0;i<N;i++){
+            int A = Integer.parseInt(reader.readLine());
+            switch (A){
+                case 0:
+                    S.pop();
+                    break;
+                default:
+                    S.push(A);
+                    break;
```

Comment:
p5 : 삼항연산자를 사용하신다면 더 깔끔할 것 같아요
파이썬 문법은 제가 잘 모르니 JS식대로 표기해본다면

```suggestion
           A ? S.pop() : S.push(A)
```

그리고 변수를 작명할 때 대문자는 피하시는게 좋을 것 같아요!
변수 작명에 대한 링크 하나 별첨드립니다

[Sooftware Coding - 좋은 코딩 습관 (네이밍)](https://sooftware.io/coding-habit/)

Reply:
의견 및 조언 감사합니다 !